keras_cv/layers/preprocessing3d/global_z_rotation.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing3d.base_augmentation_layer_3d import (
+    BaseAugmentationLayer3D,
+)
+from keras_cv.ops.point_cloud import coordinate_transform
+from keras_cv.ops.point_cloud import wrap_angle_rad
+
+POINT_CLOUDS = "point_clouds"
+BOUNDING_BOXES = "bounding_boxes"
+
+
+class GlobalZRotation(BaseAugmentationLayer3D):
```

Comment:
Should this instead be something like:  Random3DRotation(x_factor, y_factor, z_factor)?

If we only want to support Z for now, we can just raise an error when x_factor and y_factor are provided.

Reply:
While this is a good point for generalization point, I doubt we will ever support rotating on the other two axes?

Reply:
Thanks for the suggestion. Z rotation is commonly used. We could potentially support X/Y rotation in future works.

Reply:
Should we ship the API as a 3d rotation layer then?  We can simply not support X or Y rotations for the time being

Reply:
I supported  X, Y, and Z rotations and changed the class name to GlobalRandomRotation.